containers/trtl-init/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+HOSTNAME=$(hostname)
+echo "configuring $HOSTNAME certificates"
+
+CERT="/data/secret/$HOSTNAME.cert.pem"
+CHAIN="/data/secret/$HOSTNAME.chain.pem"
+
+if [[ -f $CERT && -f $CHAIN ]]; then
+    cp $CERT /data/certs/mtls_cert.pem
+    cp $CHAIN /data/certs/mtls_chain.pem
+    echo "mtls certificates configured"
+    exit 0
+else
+    echo "could not find certificate and/or chain for configuration"
+    exit 1
+fi
```

Comment:
This script runs before a container is booted in an [init container](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/) and copies the correct certificates, e.g. `ninja-1` certs into the `/data/certs` volume so that the pods only have access to their correct certs. Unfortunately it means the init-container has access to all the certs, but there isn't any getting around that.